package.json

```diff
@@ -22,12 +22,12 @@
   },
   "homepage": "https://github.com/seegno/angular-oauth2",
   "dependencies": {
-    "angular": "^1.4.0",
-    "angular-cookies": "^1.4.0",
+    "angular": "1.5.9",
```

Comment:
If we use the `^` notation then npm install will result in version `1.6.4` being installed, which is the latest version. Since version `1.6.0` people started reporting problems with unhandled rejections in the framework and we have exactly that problem when running the test suite. There are 6 tests failing with the latest angular version that fail due to an unhandled rejection, this does not happen when using `1.5.9`.